t/lib/t/MusicBrainz/Server/Controller/Edit/Show.pm

```diff
@@ -9,103 +9,132 @@ use MusicBrainz::Server::Test qw( html_ok );
 
 with 't::Mechanize', 't::Context';
 
+=head2 Test description
+
+This test checks whether the shared components of edit display pages work as
+expected for all kinds of editors: default / author / beginner / logged out.
+
+=cut
+
 test 'Check edit page displays basic components' => sub {
     my $test = shift;
+    my $mech = $test->mech;
     my $edit = prepare($test);
 
-    $test->mech->get_ok('/login');
-    $test->mech->submit_form( with_fields => {
+    $mech->get_ok('/login');
+    $mech->submit_form( with_fields => {
         username => 'editor2',
         password => 'pass',
     } );
 
-    $test->mech->get_ok('/edit/' . $edit->id, 'fetch edit page');
-    html_ok($test->mech->content);
+    $mech->get_ok(
+        '/edit/' . $edit->id,
+        'Fetched edit page as someone else than the edit author',
+    );
+    html_ok($mech->content);
 
-    $test->mech->content_contains(
+    $mech->content_contains(
         'Accept upon closing',
-        'mentions expire action',
+        'The edit page mentions what will happen on expiration',
     );
-    $test->mech->content_contains(
+    $mech->content_contains(
         '3 unanimous votes',
-        'mentions vote conditions',
+        'The edit page mentions the votes needed to close the edit early',
     );
-    $test->mech->content_contains(
+    $mech->content_contains(
         'Submit vote and note',
-        'contains voting button',
+        'The edit page contains the button to vote and add an edit note',
     );
-    $test->mech->content_contains(
+    $mech->content_contains(
         'Edit by <a href="/user/editor1">',
-        'mentions edit editor',
+        'The edit page lists the editor who entered the edit',
     )
 };
 
 test 'Check edit page differences for own edit' => sub {
     my $test = shift;
+    my $mech = $test->mech;
     my $edit = prepare($test);
 
-    $test->mech->get_ok('/login');
-    $test->mech->submit_form( with_fields => {
+    $mech->get_ok('/login');
+    $mech->submit_form( with_fields => {
         username => 'editor1',
         password => 'pass',
     } );
 
-    $test->mech->get_ok('/edit/' . $edit->id, 'fetch edit page');
-    html_ok($test->mech->content);
+    $mech->get_ok(
+        '/edit/' . $edit->id,
+        'Fetched edit page as the edit author',
+    );
+    html_ok($mech->content);
 
-    $test->mech->content_contains(
+    $mech->content_contains(
         'Submit note',
-        'contains note button',
+        'The edit page contains the button to add an edit note',
     );
-    $test->mech->content_lacks(
+    $mech->content_lacks(
         'You are not currently able to vote on this edit',
-        'cannot vote message is not present',
+        'The "you cannot vote" message is not present',
```

Comment:
Message in double quotes is expected to be an actual message, but it isn’t. Maybe rephrase not to use quotes?